jax/_src/lib/xla_bridge.py

```diff
@@ -58,6 +54,8 @@
 
 FLAGS = flags.FLAGS
 
+logger = util.get_logger()
```

Comment:
This one needs fixing still.

Reply:
I see, shouldn't flake8/mypy have caught this? It was all green on my local machine, mysterious.

Reply:
Fixed.